contrib/ci/generate_docker.py

```
@@ -79,12 +79,16 @@ def get_container_cmd():
 
 # insert dependencies to install
 if CROSS:
-    deps = parse_dependencies(DISTRO, ARCH_TO_DEPS_MAP[CROSS], False, cross=True)
-    deps += [f"crossbuild-essential-{CROSS}"]
+    deps_parsed, _ = parse_dependencies(
+        DISTRO, ARCH_TO_DEPS_MAP[CROSS], False, cross=True
+    )
+    deps = deps_parsed + [f"crossbuild-essential-{CROSS}"]
```

Ruff (0.15.20)

[warning] Consider `[*deps_parsed, f"crossbuild-essential-{CROSS}"]` instead of concatenation

Replace with `[*deps_parsed, f"crossbuild-essential-{CROSS}"]`

(RUF005)